job-server/src/main/scala/spark/jobserver/AkkaClusterSupervisorActor.scala

```diff
@@ -215,10 +113,12 @@ class AkkaClusterSupervisorActor(daoActor: ActorRef) extends InstrumentedActor {
     }
 
     val pb = Process(cmdString)
-    val pio = new ProcessIO(_ => (),
-                        stdout => scala.io.Source.fromInputStream(stdout)
-                          .getLines.foreach(println),
-                        stderr => scala.io.Source.fromInputStream(stderr).getLines().foreach(println))
+    val pio = new ProcessIO(
+      (_) => (),
```

Comment:
I could be wrong but the standard I've seen in lots of Scala code is:

    (_.blah(...))   // Use parentheses for simple expressions without => or when using underscore

    { v => v.doSomething(...) }    // Braces for formal closures with =>

I'm not really feeling religious about it, maybe it's just what I'm used to....